how-to-guides/celestia-app.md

```diff
@@ -140,11 +140,48 @@ required for P2P connections for all node types.
 
 The following ports are used by Celestia app nodes:
 
-| Port  | Protocol | Address   | Description | Enabled by default on node     | Flag                    |
-| ----- | -------- | --------- | ----------- | ------------------------------ | ----------------------- |
-| 2121  | TCP/UDP  | 127.0.0.1 | P2P         | true                           | N/A                     |
-| 9090  | HTTP     | 0.0.0.0   | gRPC        | true                           | `--grpc.address string` |
-| 26657 | TCP      | 127.0.0.1 | RPC         | false (only open to localhost) | `--rpc.laddr string`    |
+| Port  | Protocol | Address   | Description                                                     | Enabled by default on node     | Flag                    |
+| ----- | -------- | --------- | --------------------------------------------------------------- | ------------------------------ | ----------------------- |
+| 1317  | HTTP     | 0.0.0.0   | REST API (gRPC-gateway) - HTTP endpoints for blockchain queries | true                           | `--api.address string`  |
+| 2121  | TCP/UDP  | 127.0.0.1 | P2P networking - Communication between nodes in the network     | true                           | N/A                     |
+| 9090  | HTTP     | 0.0.0.0   | gRPC server - Programmatic access for advanced integrations     | true                           | `--grpc.address string` |
+| 26657 | TCP      | 127.0.0.1 | Tendermint RPC - JSON-RPC server for blockchain queries and txs | false (only open to localhost) | `--rpc.laddr string`    |
+
+## Understanding port exposure and security
+
+### External exposure requirements
+
+When setting up your consensus node, understanding which ports need external
+access is crucial for both functionality and security:
+
+**Ports that typically need external exposure:**
+
+- **Port 2121**: Required for P2P connectivity with other nodes in the network
+- **Port 9090**: Needed if your node will serve gRPC clients (like bridge nodes)
```

Comment:
I don't think this should be exposed globally, that would allow anyone on the internet to call the gRPC.

Reply:
can you confirm this please @rootulp ?

Reply:
Presumably an operator that runs two nodes on different machines would be sophisticated enough to assign things like static IPs to them, so only the static IP of the bridge node machine would be listened to. Something like that. Listening globally will allow such a bridge node machine to connect, sure, but would also allow any other machine in the world to connect. Which is bad.